nfs_afs_sync.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# rsync the files in the deployed directory to afs to be picked up by acr
+LOG="$(readlink -f "$(dirname -- "${BASH_SOURCE[0]}")")/sync_log.txt"
+SOURCE="/cds/group/pcds/epics-dev/screens/pydm/pmps-ui"
```

Comment:
Just so I understand: it always copies from the `epics-dev` directory, but logs to the checked-out directory (which may not be the same as the previous)? Is that intentional?

Reply:
it's intentional but I'm not sure if it's justified

Reply:
I think I need to switch the source to be current repo, else this ends up being a confusing script to use and debug later

Reply:
The other option is - which kinda made sense in my head - is always copying the deployed version and always updating the deployd version's log. That is to say, hard-coding both paths (instead of neither).

Entirely your call, of course. Just my 2c.